VndbSharp/Models/VndbFlags.cs

```diff
@@ -93,25 +93,52 @@ public enum VndbFlags
 		[FlagIdentity("staff")]
 		Staff = 1 << 11,
 
+	    /// <summary>
+	    ///		<para>Provides the list of voice actresses who worked on the VN</para>
+	    ///		<para>Valid on: Character, Staff</para>
+	    /// </summary>
+		[FlagIdentity("voiced")]
+		Voiced = 1 << 12,
+
+	    /// <summary>
+	    ///		<para>Provides a list of VNs this staff has worked on</para>
+	    ///		<para>Valid on: Staff</para>
+	    /// </summary>
+	    [FlagIdentity("vns")]
+	    VisualNovels = 1 << 13,
```

Comment:
Use the work around used for characters, having two "VisualNovel(s)" flags is ambiguous, and has purposely been avoided.